src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.styles.ts

```diff
@@ -61,6 +61,9 @@ export const styles = {
     '& button': {
       gap: '12px',
       width: '100%'
+    },
+    '& span': {
+      width: '100%'
```

Comment:
In this case span and button in one Box with styles.functionalButtons
As I wraped AppButton into <span> I've to fix width of this <span> as a childe of Box 
If I dont't wrap AbbButton into <span>, ref from button doesn't pass to Tooltip and styles of this button are broken because of tooltip element

I don't know how explaine it in another way, but it also an example froь official MUI documentation for using toltip with disable button

Reply:
If a better solution can't be found, we can keep with your one